src/edu/csus/ecs/pc2/clics/API202306/CLICSRun.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Set;
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.JudgementRecord;
+import edu.csus.ecs.pc2.core.model.Run;
+import edu.csus.ecs.pc2.core.util.IJSONTool;
+
+/**
+ * CLICS Judgment Run
+ * Contains information about a Run
+ *
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("rtFilter")
+public class CLICSRun {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String submission_id;
```

Comment:
There is no property called `submission_id` in the CLICS 2023-06 spec under "Properties of run objects".  Is this a substitution for the `judgement_id` property?  (I don't think `submission_id` and `judgement_id` are synonymous properties... are they?)

Reply:
See Above

Reply:
Closing (resolving) this comment because it is clearly unrelated to the real issue, which is the name of the class and what it corresponds to in CLICS.